client/js/drawModule.js

```diff
@@ -92,6 +92,11 @@ var drawModule = (function() {
 
   // create a restaurant mark and add it to the cluster.
```

Comment:
We did not adopt the JSDoc in client folder because in #86 we were requested to comment the code for learning purposes instead of documentation purposes. The main goal here is to add a general description in every module and complex function, as well as to add comments to most complex code blocks. These comments were accepted in #117.